package.json

```diff
@@ -6,8 +6,8 @@
   "scripts": {
     "dev": "vite",
     "build": "tsc && vite build && yarn build:sidecar",
-    "build:sidecar": "tsc --module nodenext --moduleResolution nodenext --esModuleInterop false --target esnext src/node/sidecar.ts --outDir dist/",
-    "watch:build": "vite build --watch",
+    "build:sidecar": "tsc --module nodenext --moduleResolution nodenext --esModuleInterop false --target esnext src/node/sidecar.ts --outDir node/",
+    "build:watch": "vite build --watch",
```

Comment:
fwiw the reason we do these under watch: is because you often want to run "build:*" and putting a diff env-build style in there complicates things in what inevtiably always becomes a monorepo